tests/std/tests/P0896R4_views_filter_iterator/test.cpp

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <ranges>
+#include <type_traits>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+#if _ITERATOR_DEBUG_LEVEL == 0
+#define NOEXCEPT_IDL0(...) noexcept(__VA_ARGS__)
+#else
+#define NOEXCEPT_IDL0(...) true
+#endif // _ITERATOR_DEBUG_LEVEL == 0
+
+constexpr auto is_even = [](const auto& x) { return x % 2 == 0; };

```

Comment:
Why are we adding all of this junk when we only need to test that `base() const&` works?

Reply:
obsolete as moved back to the main test